packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/vue/localsignout.md

```diff
@@ -0,0 +1,31 @@
+Components displayed via the `router-view` can access properties of that parent component, including the 'authenticated' prop added we added.
```

Comment:
Components **that appear** through the `router-view` can access properties of that parent component, including the `authenticated` prop **that we added**.

It's appear, rather than displayed. Display/displayed is transitive and requires an object to complete meaning...."'router-view' displays the component" vs  "the component that appears"

also - we are avoiding the use of "via" when possible. :)